api/cloudcontroller/ccv3/service_instance_test.go

```diff
@@ -645,6 +645,85 @@ var _ = Describe("Service Instance", func() {
 			})
 		})
 
+		Describe("UnshareServiceInstanceFromSpace", func() {
+			var (
+				serviceInstanceGUID string
+				spaceGUID           string
+
+				warnings   Warnings
+				executeErr error
+			)
+
+			BeforeEach(func() {
+				serviceInstanceGUID = "some-service-instance-guid"
+				spaceGUID = "some-space-guid"
+			})
+
+			JustBeforeEach(func() {
+				warnings, executeErr = client.UnshareServiceInstanceFromSpace(serviceInstanceGUID, spaceGUID)
+			})
+
+			When("no errors occur deleting the shared space relationship", func() {
+				BeforeEach(func() {
+					server.AppendHandlers(
```

Comment:
Another way to test this is to use the `FakeRequester` like this: https://github.com/cloudfoundry/cli/blob/167262b87a4c9dc5341dcfb049dba027748d25d6/api/cloudcontroller/ccv3/service_instance_test.go#L504